fixtures.ttar

```diff
@@ -355,33 +358,72 @@ xpc 399724544 92823103 86219234
 debug 0
 Mode: 644
 # ttar - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+Path: fixtures/proc/invalid/mdstat
```

Comment:
I'd prefer to keep the contents of `fixtures/proc` as close to a real system as possible.  Can you move this test example in-line with the test sources as a multi-line string and then call `parseMDStat` directly?

Reply:
Sure! Onto it